CONTRIBUTING.md

```diff
@@ -43,6 +43,17 @@ a "soft" commitment.
 If you're eager to increase the velocity of the dev process, reviewing other contributors work is
 the best you can do while waiting review on yours.
 
+Getting Project Context
+-----------------------
+
+Browsing through the [meeting minutes](https://github.com/rust-bitcoin/rust-lightning/wiki/Meetings)
+is a good first step. You will learn who is working on what, how releases are drafed, what are the
+pending tasks to deliver, where you can contribute review bandwidth, ...
+
+A secondary step might to watch this talk on the LDK project (TODO: Val's Advancing Bitcoin).
```

Comment:
@valentinewallace do you think we can link your Advancing Bitcoin presentation ? AFAIK that's the best video presenting LDK and it's not linked anywhere else ?

Reply:
Found the link: https://vimeo.com/showcase/7131712/video/418412286

Reply:
I did remove this line from latest push, let's keep it for the LDK-website v0 or link it directly in README.md as you were suggesting yesterday during meeting ?

Reply:
Sure, sounds good